flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/PlannerCallProcedureOperation.java

```diff
@@ -237,6 +249,14 @@ private TableResultInternal procedureResultToTableResult(
             tableResultType = DataTypes.ROW(DataTypes.FIELD("result", tableResultType));
         }
 
+        RowRowConverter rowConverter = null;
+        // if the output is struct type,
+        // we need a row converter to convert the struct value to Row
```

Comment:
The comments here are a bit confusing. Does RowRowConverter convert RowData and Row, not Struct value?

Reply:
Thx, I have refined the comment. First convert the struct value to RowData, and then use the RowRowConverter convert the RowData Row. Hope it can be clear.